detect_secrets/filters/heuristic.py

```diff
@@ -191,3 +191,11 @@ def is_lock_file(filename: str) -> bool:
         'Podfile.lock',
         'yarn.lock',
     }
+
+
+def is_swagger_file(filename: str) -> bool:
+    """
+    Filters swagger files and paths, like swagger-ui.html or /swagger/.
+    """
+    regex = re.compile(r'.*swagger.*')
```

Comment:
Let's cache this regex, as described in [Best Practices for Writing Filters](https://github.com/Yelp/detect-secrets/blob/master/docs/filters.md#best-practices). An example is found in that documentation too (search `Example #2`)

Reply:
Okay, It's done!